iothub/device/tests/Edge/EdgeModuleClientHelperTest.cs

```diff
@@ -247,108 +247,5 @@ public async Task<IotHubModuleClient> CreateMqttModuleClient()
 
             return dc;
         }
-
-        [TestMethod]
-        public async Task ModuleClient_SetReceiveCallbackAsync_SetCallback_Mqtt()
```

Comment:
Deleted?

Reply:
This method is no longer so complex that it needs a unit test. Same with the other unit tests deleted